src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -735,23 +772,9 @@ PaError PaPulseAudio_StartStreamCb( PaStream * s )
             }
             PaPulseAudio_UnLock( pulseaudioHostApi->mainloop );
 
-            for( waitLoop = 0; waitLoop < 100; waitLoop ++ )
+            if( _PaPulseAudio_WaitStreamState( pulseaudioHostApi->mainloop, stream->outputStream ) != paNoError )
```

Comment:
```suggestion
            if( _PaPulseAudio_WaitStreamState( pulseaudioHostApi->mainloop, stream->inputStream ) != paNoError )
```
This is the setup and wait path for inputStream, not outputStream.